extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -69,11 +69,26 @@ oppia.directive('imageWithRegionsEditor', [
         $scope.rectY = 0;
         $scope.rectWidth = 0;
         $scope.rectHeight = 0;
-
         // Is user currently drawing a new region?
         $scope.userIsCurrentlyDrawing = false;
         // Is user currently dragging an existing region?
         $scope.userIsCurrentlyDragging = false;
+        // Is user currently resizing an existing region?
+        $scope.userIsCurrentlyResizing = false;
+        // The horizontal direction along which user resize occurs
+        // 1 -> Left -1 -> Right 0 -> No resize
+        $scope.xDirection = 0;
+        // The vertical direction along which user resize occurs
+        // 1 -> Top -1 -> Bottom 0 -> No resize
+        $scope.yDirection = 0;
+        // Flags to check whether the direction changes while resizing
+        $scope.yDirectionToggled = false;
+        $scope.xDirectionToggled = false;
+        // A boolean that is set whenever the cursor moves out of the
+        // rectangular region while resizing.
+        $scope.outOfRegion = false;
+        // The region along borders which when hovered provides resize cursor
+        $scope.resizeRegionBorder = 10;
```

Comment:
I don't understand the comment or naming here. The name sounds like a function; I wouldn't expect this variable to be a number.

Do you mean something like resizableBorderWidthPx? (Include units.)